### ISSUE TITLE
GFF file downloaded from NCBI not recognized as GFF3 format

### ISSUE THREAD

**Issue by erinyoung** [andersen-lab/ivar#23]
I was using ivar to call variants in a bam file and used a GFF file downloaded from NCBI for feature information.

My command (with my sample bam and using [MN908947.3](https://www.ncbi.nlm.nih.gov/genome/?term=MN908947.3) as a reference with corresponding gff file from NCBI:
```
samtools mpileup -A -d 600000 -B -Q 0 sample.primertrim.sorted.bam | ivar variants -p sample.variants -q 20 -t 0 -r MN908947.reference.fasta -g GCF_009858895.2_ASM985889v3_genomic.gff
```

The command executed, but I got the following output:
```
Tue Mar 31 16:44:33 MDT 2020
[mpileup] 1 samples in 1 input files
GFF file is not in GFF3 file format!
GFF file is not in GFF3 file format!
GFF file is not in GFF3 file format!
GFF file is not in GFF3 file format!
```

OS: Centos7
ivar version: 1.1 (conda install)

**Comment by gkarthik** [comment]
Hey, thank you for bringing this to my attention. I 👎 forgot to account for lines starting with a single `#` for human readable comments. The GFF3 file from NCBI has 4 lines starting with `#!` and those lines are showing the error. It's a simple fix and this should be done for next release. Irrespective of that though, `ivar` should proceed with translation as expected. 

Another thing to note, is if a reference is not supplied to mpileup then any indels above the threshold frequency will show up as "+NN" or "-N" since mpileup doesn't have the reference to compute the cases. It is annoying to supply reference twice but it is required to do amino acid translation at least for now. 

```
samtools mpileup -A -d 600000 -B -Q 0 --reference MN908947.reference.fasta sample.primertrim.sorted.bam | ivar variants -p sample.variants -q 20 -t 0 -r MN908947.reference.fasta -g GCF_009858895.2_ASM985889v3_genomic.gff
```

**Comment by gkarthik** [comment]
Fixed by 8c370338672a0aefc7afd0825b920f623b519723
